ibm_cloud_sdk_core/utils.py

```diff
@@ -409,6 +410,36 @@ def is_json_mimetype(mimetype: str) -> bool:
         mimetype: The mimetype to check.
 
     Returns:
-        true if mimetype is a JSON-line mimetype, false otherwise.
+        true if mimetype is a JSON-like mimetype, false otherwise.
     """
     return mimetype is not None and json_mimetype_pattern.match(mimetype) is not None
+
+
+def is_text_mimetype(mimetype: str) -> bool:
+    """Returns true if 'mimetype' is a text-like mimetype, false otherwise.
+
+    Args:
+        mimetype: The mimetype to check.
+
+    Returns:
+        true if mimetype is a text-like mimetype, false otherwise.
+    """
+    return mimetype is not None and mimetype.startswith('text/')
+
+
+class TextResult(Response):
+    """Helper class to add better support for text responses.
+
+    Keyword Args:
+        response: The response to the service request.
+    """
+
+    def __init__(self, response: Response) -> None:
+        # Init parent.
+        super().__init__()
+        self.__dict__.update(response.__dict__)
+        # Overrides parent's `__repr__` method.
+        self.__repr__ = self.__repr__
```

Comment:
We need to copy all the parent attributes then overwrite the `__repr__` method to return our custom value.